RELEASE.md

```diff
@@ -0,0 +1,69 @@
+# How to make a release
+
+`traefik-proxy` is a package [available on
+PyPI](https://pypi.org/project/jupyterhub-traefik-proxy/).
+The PyPI release is done automatically by TravisCI when a tag
+is pushed.
+
+For you to follow along according to these instructions, you need
+to have push rights to the [traefik-proxy GitHub
+repository](https://github.com/jupyterhub/traefik-proxy).
+
+## Steps to make a release
+
+1. Checkout master and make sure it is up to date.
+
+   ```shell
+   ORIGIN=${ORIGIN:-origin} # set to the canonical remote, e.g. 'upstream' if 'origin' is not the official repo
+   git checkout master
+   git fetch $ORIGIN master
+   git reset --hard $ORIGIN/master
+   # WARNING! This next command deletes any untracked files in the repo
+   git clean -xfd
+   ```
+
+1. Update [changelog.rst](docs/source/changelog.rst) and add it to
+   the working tree.
+   
+   ```shell
+   git add traefik-proxy/docs/source/changelog.rst
+   ```
+
+   Tip: Identifying the changes can be made easier with the help of the
+   [choldgraf/github-activity](https://github.com/choldgraf/github-activity)
+   utility.
+
+1. Set a shell variable to be the new version you want to release.
+   The actual project version will be set from git automatically by versioneer.
```

Comment:
I would appreciate a bit more detail about what versioneer does, perhaps referenced or written out here.

Confusion point for me:
1. Do I need to run versioneer myself, in order to automatically set the project version?
   I think the answer is no, because it is part of setup.py's invocation.
2. Is versioneer as we use it a local in-repo copy of [this versioneer project](https://github.com/warner/python-versioneer) summarized into the versioneer.py file next to our setup.py file?
3. How is the version determined? Is it by relating to the latest in branch git tag referencing a commit in the branch, like `git describe` does?

Reply:
With travis (or some other CI) setup to do things "on tag" the only thing that has to be done by a human to make a release called 0.7.0 is:
```
# First, tag a new release locally:
V=0.7.0; git tag -am "release $V" $V
# Push this change up to the master repository
git push origin --tags
```

For branches and other commits which aren't exactly on the tag you get a version like `0.7.0+127.gd9335cf`.

Reply:
```suggestion
   The actual project version will be detected automatically by the versioneer script by inspecting git tags, and this script is running directly from setup.py when packaging is occurring.
```

Something like that is what I needed.

Reply:
Yikes failed to polish this sentence fully, got to run quickly to the gym!

Reply:
@consideRatio, I tried to polish it, but I'm not sure I'm good at "polishing English".

How does this sound: *The actual project version will be detected automatically by versioneer
from git tags inspection. The versioneer script will be run by setup.py when packaging is occurring.*

I already updated the PR with this, but I will happily change it if you think is not ok.

Reply:
@GeorgianaElena I think your version is excellent!